packages/components/src/input-control/test/index.js

```diff
@@ -14,8 +14,6 @@ import { useState } from '@wordpress/element';
  */
 import BaseInputControl from '../';
 
-const setupUser = () => userEvent.setup();
```

Comment:
I figured this wrapper was just unnecessary and directly inlined `userEvent.setup()`.